jp/components/calendar.md

```diff
@@ -342,7 +342,7 @@ export class CalendarSample7Component {
     <iframe id="calendar-views-iframe" data-src='{environment:demosBaseUrl}/scheduling/calendar-views' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
 </div>
 <div>
-    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="calendar-views-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="calendar-views-iframe" data-demos-base-url="{environment:demosBaseUrl}">stackblitz で開く</button>
 </div>
 
 ### キーボード ナビゲーション
```

Comment:
Missing strings from EN file.

- <kbd>PageUp</kbd> key to move to the previous month,
- <kbd>PageDown</kbd> key to move to the next month,
- <kbd>Shift</kbd> + <kbd>PageUp</kbd> keys to move to the previous year,
- <kbd>Shift</kbd> + <kbd>PageDown</kbd> keys to move to the next year,
- <kbd>Home</kbd> key to focus the first day of the current month  or first month in view
- <kbd>End</kbd> key to focus the last day of the current month or last month in view
- <kbd>Tab</kbd> key to navigate through the subheader buttons;

When `prev` or `next` month buttons (in the subheader) are focused, use:
- <kbd>Space</kbd> or <kbd>Enter</kbd> key to scroll into view the next or previous month.

When `months` button (in the subheader) is focused, use:
- <kbd>Space</kbd> or <kbd>Enter</kbd> key to open the months view.

When `year` button (in the subheader) is focused, use:
- <kbd>Space</kbd> or <kbd>Enter</kbd> key to open the decade view.

When a day inside the current month is focused, use:
- Arrow keys to navigate through the days,
- Arrow keys to navigate to previous/next month as well,
- Navigating next from last day in current month or previous from first day in current month, will move focus to next/previous month that is in view.
- Navigating next from last day in last visible current month or previous from first day in first current month, will change the months in view.
- <kbd>Enter</kbd> key to select the currently focused day.

When a month inside the months view is focused, use:
- Arrow keys to navigate through the months,
- <kbd>Home</kbd> key to focus the first month inside the months view,
- <kbd>End</kbd> key to focus the last month inside the months view,
- <kbd>Enter</kbd> key to select the currently focused month and close the view.

When an year inside the decade view is focused, use:
- Arrow keys to navigate through the years,
- <kbd>Enter</kbd> key to select the currently focused year and close the view.